app/controllers/admin/problems_controller.rb

```diff
@@ -1,4 +1,57 @@
 # frozen_string_literal: true
 
 class Admin::ProblemsController < Admin::ApplicationController
+  before_action :set_session
+  before_action :set_mission
+  before_action :set_problem, only: %i[show]
+
+  def new
+    @problem = @mission.problems.new
+  end
+
+  def create
+    @problem = Problem.create(
+      mission: @mission,
+      name: params[:problem][:name],
+      detail: params[:problem][:detail],
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.